ofono/include/sms.h

```diff
@@ -27,6 +27,7 @@ extern "C" {
 #endif
 
 #include <ofono/types.h>
+#include <ofono/dbus.h>
```

Comment:
This include doesn't seem to be necessary.

Reply:
Dropped a31e99b0a6bb56b996c9d1e9f3bd7cf0aef2f5a0